libs/packages.yml

```diff
@@ -390,3 +390,7 @@ packages:
   repo: cvs-health/langfair
   path: .
   downloads: 0
+- name: abso
```

Comment:
i think this should be `langchain-abso` (the pypi package name)

Reply:
fixed